config.js

```diff
@@ -13,7 +13,12 @@ module.exports = require('rc')(
       verificationUrl: 'https://accounts.firefox.com/v1/verify_email',
       passwordResetUrl: 'https://accounts.firefox.com/v1/complete_reset_password',
       accountUnlockUrl: 'https://accounts.firefox.com/v1/complete_unlock_account',
-      initiatePasswordResetUrl: 'https://accounts.firefox.com/v1/reset_password'
+      initiatePasswordResetUrl: 'https://accounts.firefox.com/reset_password',
```

Comment:
I don't recall why these all have `v1` in there, but if we're dropping it from one, should we drop it from all?